docs/source/configuration/authorization.mdx

```diff
@@ -417,19 +417,18 @@ If _every_ requested field requires authentication and a request is unauthentica
 ### `@policy`
 
 The `@policy` directive marks fields and types as restricted based on authorization policies evaluated in a [Rhai script](../customizations/rhai/) or [coprocessor](../customizations/coprocessor). This enables custom authorization validation beyond authentication and scopes.
-The directive should include a `policies` argument that defines an array of the required policies.
+
+The `@policy` directive includes a `policies` argument that defines an array of the required policies. The following example shows a policy that requires all roles from the claims object to contain the string `"support"`:
 
 ```graphql
 @policy(policies: [[]"claims[`roles`].contains(`support`)"]])
 ```
 
-The preceding example includes one policy that requires all roles from the claims object to include the string `"support"`.
-
-The Apollo Router extracts the list of policies relevant to a request from the schema. It then stores them in the request's context in `apollo_authorization::policies::required` as a map `policy -> null|true|false`. This process happens at the [`RouterService` level](../customizations/overview#the-request-lifecycle).
+At the [`RouterService` level](../customizations/overview#the-request-lifecycle), the Apollo Router extracts the list of policies relevant to a request from the schema and then stores them in the request's context in `apollo_authorization::policies::required` as a map `policy -> null|true|false`.
 
-You need to provide a Rhai script or a coprocessor at the `SupergraphService` level that evaluates the map, setting the value to `true` if the policy is validated and `false` if not. After that, the router filters the requests' types and fields to only those where the policy is `true`.
+At the `SupergraphService` level, you need to provide a Rhai script or a coprocessor that evaluates the map. If the policy is validated, set its value to `true`, otherwise set it to `false`. Afterwards, the router filters the requests' types and fields to only those where the policy is `true`.
```

Comment:
```suggestion
At the `SupergraphService` level, you must provide a Rhai script or coprocessor to evaluate the map. If the policy is validated, the script or coprocessor should set its value to `true` or otherwise set it to `false`. Afterward, the router filters the requests' types and fields to only those where the policy is `true`.
```

Reply:
What happens if the policy is still `null` (e.g., because it's not implemented in the coprocessor yet)?